src/supermarket/app/views/pages/documentation.html.erb

```diff
@@ -11,7 +11,7 @@
       <li><a href="https://www.chef.io/blog/2015/03/16/using-chef-supermarket-a-guided-tour/">Using Chef Supermarket: A Guided Tour</a></li>
       <li><a href="https://www.chef.io/blog/2015/05/15/public-supermarket-the-insiders-tour/">Public Supermarket: The Insider's Tour</a></li>
       <li><a href="https://www.chef.io/blog/2015/12/31/a-supermarket-of-your-own-running-a-private-supermarket/">A Supermarket of Your Own: Running a Private Supermarket</a></li>
-      <li><a href="https://docs.chef.io/supermarket.html">Supermarket Docs</a></li>
+      <li><a href="https://docs.chef.io/supermarket">Supermarket Docs</a></li>
```

Comment:
Should this use that same chef_docs_url helper?

Reply:
yes, it can use chef_docs_url also.. I will make the change thanks